sdk/src/Services/DynamoDBv2/Custom/DataModel/Utils.cs

```diff
@@ -109,7 +174,17 @@ public static Type GetPrimitiveElementType(Type collectionType)
 
             throw new InvalidOperationException("Unable to determine element type");
         }
-        public static Type GetElementType(Type collectionType)
+
+#if NET8_0_OR_GREATER
+        [System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessage("ReflectionAnalysis", "IL2073",
```

Comment:
I couldn't figure out how to remove the trim warning for the `collectionType.GetElementType()` type. I'm confident though at this point in using the library that type will already be known. The type would have been referenced in the application code.

To give me extra confidence that nobody was using these utilities methods out of context I changed all the visibility of these utility methods to `internal`.